plugin/gentoo-common.vim

```diff
@@ -56,8 +56,33 @@ fun! GentooGetPythonTargets()
     if exists("g:gentoopythontargets") && g:gentoopythontargets != ""
         return g:gentoopythontargets
     else
-        let l:py3 = system("python -c 'import epython; print(epython.EPYTHON)'")
-        let l:py3 = substitute(l:py3, "\n", "", "g")
+        let l:pyexec_path = "/etc/python-exec/python-exec.conf"
+
+        if filereadable(l:pyexec_path)
+            let l:pys = readfile(l:pyexec_path)->filter("v:val =~ '^[^#-].*'")
+            let l:py3s = []
+            let l:others = []
+            for l:py in l:pys
+                let l:m = l:py->matchstr("^python3.*")->matchstr("\\d*$")
+                if !empty(l:m)
+                    eval l:py3s->add(l:m)
+                    continue
+                endif
+                let l:m = l:py->matchstr("^pypy.*")
+                if !empty(l:m)
+                    eval l:others->add(l:m)
+                    continue
+                endif
```

Comment:
Do you actually need to special-case PyPy, given it would fall into `l:others` anyway?

Reply:
Not necessary, and this is atcually artefact of pre-push version, where `pypy` had separate list. That guaranteed that `pypy`s would be right after `python`s in `COMPAT` list. Without that the list after pythons would be sorted alphbetically.

What behaviour would be prefered?
* Pypy's before other "others", or
* it is only make sense to have pythons be first, and then - it is ok to have alphabetically sorted list?

Reply:
No special treatment of PyPy is necessary, plus we don't really support anything else at the moment ;-).